packages/project-access/test/project/cap.test.ts

```diff
@@ -173,3 +174,45 @@ describe('Test getCapEnvironment', () => {
         expect(forSpy).toHaveBeenCalledWith('cds', 'PROJECT_ROOT');
     });
 });
+
+describe('toReferenceUri', () => {
+    beforeEach(async () => {
+        jest.restoreAllMocks();
+    });
+    test('toReferenceUri with refUri starting with "../"', async () => {
+        // mock reading of package json in root folder of sibling project
+        const packageJson = '{"name": "@capire/bookshop"}';
+        const buf = Buffer.from(packageJson, 'utf-8');
+        jest.spyOn(file, 'readFile').mockImplementation(async (uri) => {
+            return uri ===
+                (os.platform() === 'win32'
+                    ? '\\globalRoot\\monoRepo\\bookshop\\package.json'
+                    : '/globalRoot/monoRepo/bookshop/package.json')
+                ? buf.toString()
+                : '';
+        });
+        // prepare
+        const projectRoot = '/globalRoot/monoRepo/fiori'.split('/').join(sep); // root folder of fiori project within monorepo
+        const relUriFrom = './app/admin/fiori.cds'.split('/').join(sep); // relative (to project root) uri of file for which the using statement should be generated
+        const relUriTo = '../bookshop/srv/admin.cds'.split('/').join(sep); // relative (to project root) Uri to the file that would be referenced with using statement (goes to sibling project in monorepo)
+        // execute
+        const refUri = await toReferenceUri(projectRoot, relUriFrom, relUriTo);
```

Comment:
Would not `join()` without `.split('/').join(sep)` do the same:

```suggestion
        const projectRoot = join('/globalRoot/monoRepo/fiori'); // root folder of fiori project within monorepo
        const relUriFrom = join('./app/admin/fiori.cds'); // relative (to project root) uri of file for which the using statement should be generated
        const relUriTo = join('../bookshop/srv/admin.cds'); // relative (to project root) Uri to the file that would be referenced with using statement (goes to sibling project in monorepo)
```

Reply:
adapted